### PR TITLE
pedantry

### PR DESCRIPTION
**Pull request by zeke**
screen shot 2014-08-27 at 11 41 02 pm

### CODE REVIEW

**Comment by yields** [comment]
haha lgtm!

**Comment by matthewmueller** [comment]
oh man, @ianstormtaylor had it right and I changed it. learned something new today!

**Comment by zeke** [comment]
:cake:

**Comment by ianstormtaylor** [comment]
i'm nothing if not pedantic haha

File: Readme.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The [duojs.org](http://duojs.org) website.
 
 ## Developing
 
+Make sure you're running Node version `0.11` or greater.
```
Comment (stephenmathieson) [inline Readme.md:15]: not true -- we use `gnode` so node 10 works fine

**Pull request merged**